library/src/main/java/co/moonmonkeylabs/realmrecyclerview/RealmRecyclerView.java

```diff
@@ -75,6 +79,15 @@ public RealmRecyclerView(Context context, AttributeSet attrs, int defStyleAttr)
         init(context, attrs);
     }
 
+    @Override
+    protected void onMeasure(int widthSpec, int heightSpec) {
+        super.onMeasure(widthSpec, heightSpec);
+        if (gridWidth != -1 && gridManager != null) {
```

Comment:
This only needs to happen when `getMeasueredWidth` is different than `gridWidth`.